pages/index.js

```diff
@@ -31,6 +31,19 @@ export async function getServerSideProps(context) {
   };
 }
 
+//will change the location value in the selector to be whatever the geolocation API returns
+//also returns the canadian province or "Other" the api acquires
+async function assumeLocation() {
+  //Geolocation API Key, should probably be moved to some sort of global thing
+  const apiKey = "c81cd9865ea747f580d359ee1758d5be";
```

Comment:
This looks to be a secret. Typically would live in .env and we need to do some work in team city. Ask Kyle or myself for some help with team city